modules/openapi-generator/src/main/resources/C-libcurl/api-body.mustache

```diff
@@ -11,6 +11,91 @@
     snprintf(dst, 256, "%ld", (long int)(src));\
 }while(0)
 
+{{#operations}}
+{{#operation}}
+{{#allParams}}
+
+{{#isEnum}}
+// Functions for enum {{enumName}} for {{{classname}}}_{{{operationId}}}
+
+
+char* {{enumName}}_ToString({{baseName}}_e {{enumName}}){
+char *{{enumName}}Array[] =  { {{#allowableValues}}{{#enumVars}}"{{{value}}}"{{^-last}},{{/-last}}{{/enumVars}}{{/allowableValues}} };
+    return {{enumName}}Array[{{enumName}}];
+}
+
+{{baseName}}_e {{enumName}}_FromString(char* {{enumName}}){
+    int stringToReturn = 0;
+    char *{{enumName}}Array[] =  { {{#allowableValues}}{{#enumVars}}"{{{value}}}"{{^-last}},{{/-last}}{{/enumVars}}{{/allowableValues}} };
+    size_t sizeofArray = sizeof({{enumName}}Array) / sizeof({{enumName}}Array[0]);
+    while(stringToReturn < sizeofArray) {
+        if(strcmp({{enumName}}, {{enumName}}Array[stringToReturn]) == 0) {
+            return stringToReturn;
+        }
+        stringToReturn++;
+    }
+    return 0;
+}
+
+cJSON *{{enumName}}_convertToJSON({{baseName}}_e {{enumName}}) {
+cJSON *item = cJSON_CreateObject();
+{{#isString}}
+    if(cJSON_AddStringToObject(item, "{{{paramName}}}", {{{enumName}}}_ToString({{{enumName}}})) == NULL) {
+        goto fail;
+    }
+{{/isString}}
+{{#isNumeric}}
+    if(cJSON_AddNumberToObject(item, "{{{paramName}}}", {{{enumName}}}) == NULL) {
+        goto fail;
+    }
+{{/isNumeric}}
+    return item;
+    fail:
+    cJSON_Delete(item);
+    return NULL;
+}
+
+{{baseName}}_e {{enumName}}_parseFromJSON(cJSON *{{enumName}}JSON){
+
+{{{baseName}}}_e {{enumName}}Variable = 0;
+{{#isNumeric}}
+cJSON *{{{enumName}}}Var = cJSON_GetObjectItemCaseSensitive({{enumName}}JSON, "{{{paramName}}}");
+if(!cJSON_IsNumber({{{enumName}}}Var))
+{
+    goto end;
+}
+{{/isNumeric}}
+{{#isString}}
+cJSON *{{{enumName}}}Var = cJSON_GetObjectItemCaseSensitive({{enumName}}JSON, "{{{paramName}}}");
+if(!cJSON_IsString({{{enumName}}}Var) || ({{{enumName}}}Var->valuestring == NULL)){
+    goto end;
+}
+{{enumName}}Variable = {{enumName}}_FromString({{{enumName}}}Var->valuestring);
+{{/isString}}
+return {{enumName}}Variable;
+end:
+return 0;
+}
+{{/isEnum}}
+
+{{/allParams}}
+{{/operation}}
+{{/operations}}
+
+
+
```

Comment:
Hello @michelealbano

There are so many blank lines here. Could you please remove them ?

Reply:
Done